pkg/cloud/awserrors/errors.go

```diff
@@ -63,26 +63,26 @@ func Message(err error) string {
 
 // EC2Error is an error exposed to users of this library.
 type EC2Error struct {
-	err error
+	err string
```

Comment:
Nit: should probably avoid use of `err` if we are dealing with strings now
```suggestion
	msg string
```